src/xata/xataFetcher.ts

```diff
@@ -73,11 +76,17 @@ export async function xataFetch<
     }
 
     if (!response.status.toString().startsWith("2")) {
+      const payload = await response.json();
+
+      if (response.status === 400 && !silentError) {
+        vscode.window.showErrorMessage(payload.message);
+      }
+
```

Comment:
This is to expose any 400 errors globally to the user (so we don't have to deal with them one by one on the codebase anymore)